index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Nabil's Blog</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="main.css">
+    </head>
+    <body>
+        <div class="chom">
+            <div class="chom-triangle"></div>
+            <div class="chom-rect"></div>
+            <div class="chom-top"></div>
+        </div>
+        <div class="head-triangle">
```

Comment:
HTML brings a lot more than `divs`, so I'd recommend exploring semantic HTML for this - maybe using a `section` here and there. :)